tests/integration/test_core.py

```diff
@@ -255,3 +265,103 @@ async def test_wordpress_plugin_installation_error(
         assert (
                 unit.workload_status == ops.model.ActiveStatus.name
         ), "status should back to active after invalid plugin removed from config"
+
+
+@pytest.mark.asyncio
+async def test_openstack_object_storage_plugin(
+        ops_test: pytest_operator.plugin.OpsTest,
+        application_name,
+        default_admin_password,
+        unit_ip_list,
+        openstack_environment
+):
+    """
+    arrange: after WordPress charm has been deployed, db relation established and openstack swift
+        server ready
+    act: update charm configuration for openstack object storage plugin
```

Comment:
I feel like either here or in the assert we should be saying we're going to upload an image to swift from each unit and confirm it's downloadable as well.

Reply:
Added, thanks!